target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -77,6 +77,10 @@ tplink,re650-v1)
 xzwifi,creativebox-v1)
 	ucidef_set_led_netdev "internet" "internet" "$boardname:blue:internet" "wan"
 	;;
+xiaomi,mi-router-ac2100)
+	ucidef_set_led_switch "wan-yellow" "WAN (yellow)" "$boardname:yellow:wan" "switch0" "0x01" "0x01"
+	ucidef_set_led_switch "wan-blue" "WAN (blue)" "$boardname:blue:wan" "switch0" "0x01"
```

Comment:
`ucidef_set_led_switch` is obsolete in DSA driver. It won't work at all.

Please take a look at [MI R3G's dts](https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_xiaomi_mir3g.dts) and use speed/netdev LED trigger instead.

Reply:
Right. Looking at the ucidef_set_led_switch function definition there seems to be no speed_mask so the resulting lines should be:
`xiaomi,mi-router-ac2100)
	ucidef_set_led_netdev "wan" "wan" "$boardname:blue:wan" "wan"
	;;`
and
`xiaomi,redmi-router-ac2100)
	ucidef_set_led_netdev "wan" "wan" "$boardname:white:wan" "wan"
	;;`

Reply:
This device doesn't have amber led at wan port. The wan gpio we defined is at fron of chassih.  And  I disabled yellow wan led until we will  findworkaround for situation @phs07 told.